python/pyspark/mllib/classification.py

```diff
@@ -173,7 +173,7 @@ def __init__(self, weights, intercept, numFeatures, numClasses):
             self._dataWithBiasSize = None
             self._weightsMatrix = None
         else:
-            self._dataWithBiasSize = self._coeff.size / (self._numClasses - 1)
+            self._dataWithBiasSize = self._coeff.size // (self._numClasses - 1)
```

Comment:
Nor this?

Reply:
I had to apply this to get past a python unit test failure.  My assumption is that the NewSparkPullRequestBuilder is on a different version of numpy than when the Spark 1.6 branch was last built.  The current python unit test failure looks like it has to do with a novel version of SciPy.

Reply:
This is already fixed in the 2.0 branch, btw.  Just was never applied to 1.6.  [SPARK-20862]

Reply:
The NewSparkPullRequestBuilder failed on python tests.  I was only able to duplicate the failure with Python 3.4 and numpy 1.12.1, which I'm guessing is the versions that NewSparkPullRequestBuilder is using.  Older and newer versions of numpy build clean either way.